setup.cfg

```diff
@@ -7,10 +7,48 @@ markers =
 # please note that the values are adjusted so that they do not cause failures
 # with existing code. if you want to change them, you should first fix all
 # flake8 failures that appear with your change.
-ignore = E122,E123,E125,E126,E127,E128,E226,E402,E722,E731,E741,
-         F401,F405,F811,
-         W504
+ignore =
+    #E122,   # continuation line missing indentation or outdented
+    #E123,   # closing bracket does not match indentation of opening bracket's line
+    #E125,   # continuation line with same indent as next logical line
+    #E126,   # continuation line over-indented for hanging indent
+    #E127,   # continuation line over-indented for visual indent
+    E128,   # continuation line under-indented for visual indent
+    E226,   # missing whitespace around arithmetic operator
+    E402,   # module level import not at top
+    #E722,   # do not use bare except
+    #E731,   # do not assign a lambda expression, use def
+    #E741,   # ambiguous variable name
+    F401,   # import unsused
+    F405,   # undefined or defined from star imports
+    #F811,   # redef of unused var
+    #W504,   # line break after binary operator
+
+per-file-ignores =
+    src/borg/platform/__init__.py:F811
+    src/borg/archive.py:E122,E125,E127,W504
+    src/borg/archiver.py:E126,E127,E722,E741,W504
+    src/borg/cache.py:E127,E722,W504
+    src/borg/fuse.py:E722,W504
+    src/borg/fuse_impl.py:F811
+    src/borg/locking.py:E722
+    src/borg/repository.py:W504
+    src/borg/crypto/key.py:E125,W504
+    src/borg/helpers/misc.py:E722
+    src/borg/helpers/msgpack.py:E127
+    src/borg/helpers/parseformat.py:E741
+    src/borg/helpers/process.py:W504
+    src/borg/testsuite/archive.py:W504
+    src/borg/testsuite/archiver.py:E722
+    src/borg/testsuite/benchmark.py:F811
+    src/borg/testsuite/crypto.py:E126,E741
+    src/borg/testsuite/helpers.py:E126,E127
+    src/borg/testsuite/locking.py:E126,E722
+    src/borg/testsuite/patterns.py:E123
+    src/borg/testsuite/platform.py:F811
+    src/borg/testsuite/shellpattern.py:E123
+
 # line length long term target: 120
-max-line-length = 255
+max-line-length = 120
```

Comment:
btw, the tests output a warning for this, telling that it should be `max_line_length`.

Reply:
ok. fixed this and finished adding all the specific files for flake8 disabled checks.  Looks like from 1.1.x to 1.2 there are 12 more offending files, so getting at least the setup.cfg change in will have minimal impact on active PRs and prevent more code from violating coding styles.